single-node/single-node.yaml

```diff
@@ -410,7 +410,9 @@ Resources:
           PropagateAtLaunch: 'true'
     DependsOn:
       - WSO2ISNodeLaunchConfigurationsuffix
-
+https://github:
+  com/aaujayasena/performance-is:
```

Comment:
should not this be WSO2 repo?

Reply:
commit the fix [1]

https://github.com/wso2/performance-is/pull/58/commits/103c6c8994497e0c93072be2ab2fe44d7786aaac